recipes/opencolorio/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+from conans import ConanFile, CMake, tools
+import os, shutil
+
+class OpenEXRConan(ConanFile):
+    name = "opencolorio"
+    description = "A color management framework for visual effects and animation."
+    license = "BSD-3-Clause"
+    homepage = "https://opencolorio.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake", "cmake_find_package"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+    
+    def requirements(self):
+        self.requires("lcms/2.11")
+        self.requires("yaml-cpp/0.6.3")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("OpenColorIO-{}".format(self.version), self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+
+        self._cmake = CMake(self)
+
+        self._cmake.definitions["OCIO_BUILD_SHARED"] = self.options.shared
+        self._cmake.definitions["OCIO_BUILD_STATIC"] = not self.options.shared
+        self._cmake.definitions["OCIO_BUILD_APPS"] = False
+        self._cmake.definitions["OCIO_BUILD_DOCS"] = False
+        self._cmake.definitions["OCIO_BUILD_TESTS"] = False
+        self._cmake.definitions["OCIO_BUILD_PYGLUE"] = False
+        self._cmake.definitions["USE_EXTERNAL_TINYXML"] = False
+        self._cmake.definitions["USE_EXTERNAL_YAML"] = True
+        self._cmake.definitions["USE_EXTERNAL_LCMS"] = True
+        self._cmake.definitions["TINYXML_OBJECT_LIB_EMBEDDED"] = True
```

Comment:
Is this using an embedded `tinyxml` library?
Would it be possible to use cci's `tinyxml2/8.0.0` recipe?

Reply:
OCIO uses old TinyXML, which had it's last release 10 years ago and is incompatible with TinyXML 2. Upcoming version of OCIO will use [expat](https://github.com/AcademySoftwareFoundation/OpenColorIO/pull/614), but for now using embedded TinyXML seems to me to be the best option.

Reply:
I understand. Maybe add a `#FIXME: tinyxml should ideally be provided by a conan recipe` comment to make this potential problem grep'able.

Reply:
@p-podsiadly please, add the tag #FIXME , so we can easily filter in the future.

Reply:
Done.